examples/pissa_finetuning/README.md

```diff
@@ -0,0 +1,73 @@
+# PiSSA: Principal Singular values and Singular vectors Adaptation
+## Introduction ([Paper](https://arxiv.org/abs/2404.02948), [code](https://github.com/GraphPKU/PiSSA))
+PiSSA initializes the LoRA adapter using the principal singular values and singular vectors. This straightforward modification allows PiSSA to converge more rapidly than LoRA and ultimately attain superior performance. Moreover, PiSSA reduces the quantization error compared to QLoRA, leading to further enhancements.
+
+## Quick Start
+### Step 1. 
+Utilize the decomposed models directly from the [Hugging Face Collections](https://huggingface.co/collections/fxmeng/pissa-661ce700721235e542a5d7a8).
+If the existing settings do not meet your needs, apply [PiSSA initialization](https://github.com/fxmeng/peft/blob/606a69279480bbdea847f4e5247804bdf7e6b898/examples/pissa_finetuning/pissa_finetuning.py#L85-L103) to a pre-trained model and save the decomposed parameters:
+
+```
+# Load an original pre-processed dodel:
+model = AutoModelForCausalLM.from_pretrained(...)
+
+# Configure the initialization method to "pissa", which may take several minutes to execute SVD on the pre-trained model:
+lora_config = LoraConfig(init_lora_weights="pissa", ...) 
+
+# Alternatively, execute fast SVD, which takes only a few seconds. The number of iterations determines the trade-off between the error and computation time:
+# lora_config = LoraConfig(init_lora_weights="pissa_niter_[number of iters]", ...) 
+
+# Perform PiSSA on the original model according to lora_config:
+peft_model = get_peft_model(model, lora_config)
+```
+To eliminate the errors introduced by Fast SVD, we have modified the computation formula for the residual matrix to $W^{res} = W - AB$. Although the calculation of $A$ and $B$ involves errors, the overall initialization error is zero.
```

Comment:
From reading this, it's not clear what `W^{res}` is and how this affects the user.